tests/test_applets/conservationTracking/testTrackingBaseDataExportAppletSerialization.py

```diff
@@ -0,0 +1,47 @@
+import os
+
+import h5py
+from lazyflow.graph import Graph
+from lazyflow.operator import Operator
+
+import ilastik
+from ilastik.applets.tracking.base.trackingBaseDataExportApplet import TrackingBaseDataExportApplet
+
+
+class OpFake(Operator):
+    def __init__(self, *args, **kwargs):
+        super(OpFake, self).__init__(*args, **kwargs)
+
+
+class TestTrackingBaseDataExportAppletSerialization(object):
+    ilastik_tests_file_path = os.path.join(os.path.split(os.path.realpath(ilastik.__file__))[0], "../tests/")
+    TEST_PROJECT_FILE = os.path.join(ilastik_tests_file_path, 'test_tracking_project.ilp')
+
+    def tearDown(self):
+        os.remove(self.TEST_PROJECT_FILE)
+
+    def testAppletSerialization(self):
+        g = Graph()
+        op = OpFake(graph=g)
+        dataExportApplet = TrackingBaseDataExportApplet(op, "Tracking Result Export")
+        dataExportSerializer = dataExportApplet.dataSerializers[0]
+
+        with h5py.File(self.TEST_PROJECT_FILE) as testProject:
+            opTrackingBaseDataExport = dataExportApplet.topLevelOperator
+            opTrackingBaseDataExport.SelectedPlugin.setValue('Fiji-MaMuT')
+            opTrackingBaseDataExport.SelectedExportSource.setValue('Plugin')
+            opTrackingBaseDataExport.AdditionalPluginArguments.setValue({'bdvFilePath': '/tmp/bdv.xml'})
+
+            dataExportSerializer.serializeToHdf5(testProject, self.TEST_PROJECT_FILE)
+
+        # check serialized values
+        with h5py.File(self.TEST_PROJECT_FILE) as testProject:
+            assert testProject["Tracking Result Export/SelectedPlugin"].value.decode() == 'Fiji-MaMuT'
+            assert testProject["Tracking Result Export/SelectedExportSource"].value.decode() == 'Plugin'
+            assert testProject["Tracking Result Export/AdditionalPluginArguments/bdvFilePath"].value.decode() == '/tmp/bdv.xml'
+
```

Comment:
testing deserialization would be really good, as we have talked about it

Reply:
Yes, it should have been added. Thanks for the reminder. It's done.
:white_check_mark: